### DISCUSSION TITLE
Node.js on Codespaces? (RESOLVED)

### DISCUSSION THREAD

**Discussion by ItsAurelai** [community/community#39098] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi!
How can i install Node.js in my Codespaces Environment?

**Comment by craiglpeters** [comment, marked as answer]
If you create a codespace from the Blank template, or most repositories that use the defaults, Node is already installed. The quickstart example uses Node https://docs.github.com/en/codespaces/getting-started/quickstart

**Reply by ItsAurelai** [reply]
Oh ok. I completely forgot to check the quick start guide. Thank you so much!
